test/controllers/incomeSources/cease/IncomeSourceEndDateControllerSpec.scala

```diff
@@ -412,7 +412,9 @@ class IncomeSourceEndDateControllerSpec extends TestSupport with MockAuthenticat
       }
     }
   }
+
   "IncomeSourceEndDateController submit/submitChange/submitAgent/submitChangeAgent" should {
```

Comment:
why are there no changes to manage businesses tests in this PR? i thought they would have broken

Reply:
surprisingly there are no broken tests in manageBusiness, might be missing UTs??